policy/modules/apps/cdrecord.te

```diff
@@ -113,3 +132,98 @@ tunable_policy(`use_nfs_home_dirs',`
 optional_policy(`
 	resmgr_stream_connect(cdrecord_t)
 ')
+
+########################################
+#
+# Cdripper (e.g. cdda2wav) local policy
+#
+
+allow cdripper_t self:capability { ipc_lock sys_nice setgid setuid dac_override sys_rawio };
+allow cdripper_t self:process { getcap getsched setcap setrlimit setsched sigkill };
+allow cdripper_t self:fifo_file rw_fifo_file_perms;
+allow cdripper_t self:tcp_socket create_socket_perms;
+allow cdripper_t self:udp_socket create_socket_perms;
```

Comment:
The above is probably redundant is it is probably already enclosed with the below `sysnet_dns_name_resolve(cdrippter_t)` call, however I would probably call `auth_use_nsswitch(cdripper_t)` instead (which by the way also implies `sysnet_dns_name_resolve(cdripper_t)`

Reply:
Partial ACK.

I prefer not to further extend the permission set by involving NSSWITCH when all is needed is a DNS lookup for the CDDB server.

But thanks for pointing out the rest, useful !